app/models.py

```diff
@@ -0,0 +1,8 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class Actor:
+    actor_id: int
```

Comment:
Rename `actor_id` to `id` to match the column name used in the SQL queries within `ActorManager` in `managers.py`.